### PR TITLE
usbredir 0.8.0 (new formula)

### PR DESCRIPTION
**Pull request by jeffreywildman**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/usbredir.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "libusb"
+
+  def install
+    # https://stackoverflow.com/questions/15860127/how-to-configure-tcp-keepalive-under-mac-os-x
+    inreplace "usbredirserver/usbredirserver.c" do |s|
+      s.gsub! "SOL_TCP", "IPPROTO_TCP"
+      s.gsub! "TCP_KEEPIDLE", "TCP_KEEPALIVE"
```
Comment (fxcoudert) [inline Formula/usbredir.rb]: This needs to be reported and fixed upstream. We ship software that builds natively on macOS.

Comment (jeffreywildman) [reply Formula/usbredir.rb]: Reported [upstream](https://gitlab.freedesktop.org/spice/usbredir/issues/9).

Comment (jeffreywildman) [reply Formula/usbredir.rb]: Patch [submitted](https://lists.freedesktop.org/archives/spice-devel/2019-January/047521.html) upstream.

Comment (jeffreywildman) [reply Formula/usbredir.rb]: Patch merged upstream, formula updated to apply upstream patch ahead of next release.

**Comment by javian** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Thanks @jeffreywildman for your contribution!

**Pull request closed without merging**